### ISSUE TITLE
[Accessibility][Forms and validation]: 'Actions' label is not associated with its corresponding control. (Bug 1)

### ISSUE THREAD

**Issue by jdhuntington** [OfficeDev/office-ui-fabric-react#10827]
Tracking all of the issues brought up with the 08/24 accessibility pass on Contextual Menu  control
Details on individual bugs can be found on [SharePoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7Bcef6201c-bd85-44b6-a2d3-3784248461e0%7D&action=default&uid=%7BCEF6201C-BD85-44B6-A2D3-3784248461E0%7D&ListItemId=1259&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble)

Breakout of issue #6360

**Comment by joschect** [comment]
Related to dropdown issue #10921 I believe that both of these will need more review and design input as to the proper way that these should get read and handled.

**Comment by micahgodbolt** [comment]
1. Open 'https://developer.microsoft.com/en-us/fabric#/components/contextualmenu' in Edge.   2. Navigate to 'ContextualMenu with section headers' section.    3. Navigate to 'Click for ContextualMenu' control using tab key.   4. Navigate to 'New' control using tab key.   5. Observe the issue. | Actions group' label is not associated with its corresponding control. | labels like: (Actions, social, Navigation) is not associted with its first element of the group.



Actions group' label is not associated with its corresponding control.



labels like: (Actions, social, Navigation) is not associted with its first element of the group.
